package.json

```diff
@@ -39,11 +40,11 @@
     "bs-react-test-renderer": "^1.0.0",
     "html-webpack-plugin": "^2.28.0",
     "jest": "^20.0.4",
-    "npm-run-all": "^4.0.2",
```

Comment:
thank you! my bad, i thought npm-run-all was the issue, removed it, and forget to put it back in 🙏